src/main/java/seedu/address/logic/parser/booking/AddBookingCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser.booking;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.util.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.util.ParserUtil.arePrefixesPresent;
+
+import seedu.address.exception.ParseException;
+import seedu.address.logic.commands.booking.AddBookingCommand;
+import seedu.address.logic.descriptors.BookingDescriptor;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.util.ArgumentMultimap;
+import seedu.address.logic.parser.util.ArgumentTokenizer;
+import seedu.address.logic.parser.util.ParserUtil;
+
+public class AddBookingCommandParser implements Parser<AddBookingCommand> {
+    @Override
+    public AddBookingCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_PHONE);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_PHONE) || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
```suggestion
        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_PHONE) || !argMultimap.getPreamble().isEmpty()) {
```
To standardise with your change to `AddCommandParser`.